### ISSUE TITLE
Accessibility issue in Chrome w/ VoiceOver not reading dropdown items

### ISSUE THREAD

**Issue by heymaddieh** [downshift-js/downshift#727]
- `downshift` version: 3.2.10
- `node` version: 10.16.0
- `npm` (or `yarn`) version: npm 6.9.0
- Chrome version:  75.0.3770.100 

**Reproduction repository**:

Replicated in Code Sandbox: https://codesandbox.io/embed/interesting-lederberg-7vmlg
Also happening in Downshift example: https://downshift.netlify.com/tests/combobox

**Problem description**:
VoiceOver is failing to read the list items text when selecting option with up/down arrows. If you navigate further into the listbox with the keyboard, it will read the options as text, but they aren't selectable using the keyboard. This only happens in Chrome. Everything is working properly in Safari.

**Comment by silviuavram** [comment]
You are using Downshift default as a dropdown when it's actually a combobox by default. Different patterns. For it to read the options:
- the combobox needs to have focus on the <input> always when opened. Because that is where we are adding `aria-activedescendant`.
- the dropdown (your case) needs to have focus on the <ul> always when opened. Same reason.

Try [this](https://github.com/silviuavram/downshift-hooks). I am experimenting with this hook. It implements the design pattern you might need.

Let me know if I can assist you further.

**Comment by heymaddieh** [comment]
@silviuavram Do you know why it would work in Safari, but not in Chrome?

**Comment by silviuavram** [comment]
No idea, but on Safari it works strange too. The VO focus is on another element, maybe a DIV somewhere, and it reads `bla bla, clickable`. If you take a look at https://downshift-hooks.netlify.com/ in any of the examples, they should work on both Chrome and Safari with VO. The reason is the fact that focus is applied on the list after button opens the menu.

**Comment by heymaddieh** [comment]
I made the change and implemented the `useSelect` hook, and this solved the problem. Thank you!

**Comment by jamesb3ll** [comment]
For future reference, if not using `useSelect` hook, this can be solved by adding `aria-activedescendant` attribute to the toggle button, we can get this from `getInputProps`.
Example:
```
<button
  type="button"
  {...getToggleButtonProps({
    "aria-activedescendant": getInputProps()["aria-activedescendant"]
  })}
>
  {this.state.value}
</button>
{!!isOpen && (
  <ul {...getMenuProps()}>
    {options.map((item, index) => (
      <li
        {...getItemProps({
          key: item,
          index,
          style: { backgroundColor: highlightedIndex === index ? "#e1f0fe" : "white" }
        })}
      >
        {item}
      </li>
    ))}
  </ul>
)}
```

*Code Sandbox:*
https://codesandbox.io/s/musing-browser-jmqdn

**Comment by silviuavram** [comment]
Going further we would like to have focus on the hooks though :)

**Comment by mimi26** [comment]
@jamesb3ll that example isn't working at all with VoiceOver in Safari, which is the browser most commonly used in combination with VoiceOver.

**Comment by cheryl-c-tirerack** [comment]
Why is this closed??
None of the comments up there actually work.
I need this with the useSelect hook.
VoiceOver does not ready any of the li's

**Comment by silviuaavram** [comment]
@cheryl-c-tirerack what is the usage you are experiencing issues with? `useSelect` as well as `useCombobox` should work fine.
